ui/wallet_page.go

```diff
@@ -40,20 +55,47 @@ func (win *Window) WalletPage(common pageCommon) layout.Widget {
 		accountsList: layout.List{
 			Axis: layout.Vertical,
 		},
-		addWallet: new(widget.Button),
-
-		beginRenameW:             common.theme.PlainIconButton(common.icons.contentCreate),
-		cancelRenameW:            common.theme.PlainIconButton(common.icons.contentClear),
-		renameW:                  common.theme.PlainIconButton(common.icons.navigationCheck),
-		editorW:                  common.theme.Editor("Enter wallet name"),
-		addAcctW:                 common.theme.IconButton(common.icons.contentAdd),
-		deleteW:                  common.theme.DangerButton("Delete Wallet"),
-		gotoSignMessagePageBtn:   win.outputs.signMessageDiag,
-		signMessageDiag:          &win.inputs.signMessageDiag,
-		gotoVerifyMessagePageBtn: win.outputs.verifyMessDiag,
-		addWalletW:               common.theme.Button("Add Wallet"),
-		verifyMessageDiag:        &win.inputs.verifyMessDiag,
+		wallet:        common.wallet,
+		line:          common.theme.Line(),
+		editorW:       common.theme.Editor("New wallet name"),
+		renameW:       common.theme.Button("Rename Wallet"),
+		errorLabel:    common.theme.Body2(""),
+		result:        &win.signatureResult,
+		deleteW:       common.theme.DangerButton("Confirm Delete Wallet"),
+		cancelDeleteW: common.theme.Button("Cancel Wallet Delete"),
+		passwordModal: common.theme.Password(),
+		errChannel:    common.errorChannels[PageWallet],
 	}
+	page.line.Color = common.theme.Color.Gray
+	page.errorLabel.Color = common.theme.Color.Danger
+
+	page.icons.addAcctW = common.theme.IconButton(common.icons.contentAdd)
+	page.icons.addAcctW.Padding = unit.Dp(5)
+	page.icons.addAcctW.Size = unit.Dp(30)
+	page.icons.mainW = common.theme.IconButton(common.icons.navigationArrowBack)
+	page.icons.mainW.Background = color.RGBA{}
+	page.icons.mainW.Color = common.theme.Color.Hint
+	page.icons.mainW.Padding = unit.Dp(0)
+	page.icons.mainW.Size = unit.Dp(30)
```

Comment:
since these are repeated and in the closure, you can declare them as variables as simply use the variable here.